### ISSUE TITLE
Error running jest with `--no-cache` after upgrading to Angular V14

### ISSUE THREAD

**Issue by Bil0** [just-jeb/angular-builders#1247]
## Describe the Bug

After upgrading to Angular V14. running tests with `no-cache` throws an error

```sh
Error: Schema validation failed with the following errors:
  Data path "/noCache" must be array.
error Command failed with exit code 1.
```

## Minimal Reproduction
here the repo: https://github.com/Bil0/angular-builders-jest-cache
`npm run test` to run ng test with `--no-cache`

## Expected Behavior
tests should run with Angular V14.

## Screenshots
![no-cache](https://user-images.githubusercontent.com/10976283/185073102-5b506cfe-4211-4be8-9629-ed3bb6f13fbc.PNG)

## Environment

<pre><code>
Libs
@angular-devkit/architect       0.1401.2
@angular-devkit/build-angular   14.1.2
@angular-devkit/core            14.1.2
@angular-devkit/schematics      14.1.2
@schematics/angular             14.1.2
rxjs                            7.5.6
typescript                      4.7.4
@angular-builders/jest    14.0.0,

For Tooling issues:
- Node version:  v16.13.1
- Platform: Windows
</code>
</pre>
## Additional Context
I think, it get mixed with the new `ng cache`  command

**Comment by Bil0** [comment]
As workaround, adding the option `no-cache: true` or `cache: false` in angular.json  worked.
```ts
"test": {
          "builder": "@angular-builders/jest:run",
          "options": {
            "no-cache": true
          }
}
```

**Comment by just-jeb** [comment]
What happens if you do `ng test --cache false`?

**Comment by Bil0** [comment]
Same error with `ng test --cache false`
![image](https://user-images.githubusercontent.com/10976283/185359457-217f5ee0-ef29-480f-9774-4048efca8df8.png)

**Comment by just-jeb** [comment]
Sorry it took so long. [This](https://github.com/just-jeb/angular-builders/pull/1281) PR is supposed to fix the issue, once it's merged and a beta is released please let me know if it fixed the issue for you.

**Comment by just-jeb** [comment]
Should be fixed in `14.0.2-beta.1`, please confirm.

**Comment by Bil0** [comment]
Hello, thanks for your response.
I updated my project to use  `14.0.2-beta.1` and I'm having the same error.

I think `no-cache` is the conflicting command and not `cache`.
tested it by editing schema.json in `node_modules`

**Comment by just-jeb** [comment]
Fixed in `14.0.2` (Release version)
